src/serving.ts

```diff
@@ -68,21 +70,7 @@ export class Server {
   constructor(public config: Config) {
     if (this.config.isProduction) {
       this.log = { write : (_s:string) :void => {} };
-    // TODO(ldixon): do the cloud logging thing...
-    // something like this...
-    //   // Instantiates a client
-    //   const loggingClient = Logging();
-    //   // The name of the log to write to
-    //   const LOG_NAME = 'convai-server-log';
-    //   // Selects the log to write to
-    //   let logWrite = loggingClient.log(LOG_NAME).write;
-    //   this.log = { write : (s:string) :void => {
-    //     // We wrap this just to catch the error so that node doesn't crash
-    //     // should stackdriver fail a log statement.
-    //     // Once we confirm that stackdriver log functions never fail their promise,
-    //     // then this can be removed.
-    //     logWrite(s).catch((e:Error) => { console.error(e); });
-    //   } };
+      // TODO(ldixon): do the cloud logging thing...
```

Comment:
You can delete this comment too now. Cloud logging changed and fixed the bad bugs I that were problematic here before.

Reply:
Done